pkg/provider/common/common.go

```diff
@@ -0,0 +1,15 @@
+package common
+
+const (
+	RetestRegex   = "(^|\\\\r\\\\n)/retest([ ]*$|$|\\\\r\\\\n)"
```

Comment:
@vdemeester would get a heart attack if he sees the word "common" anywhere in the code :-D 

you can just have it in pkg/provider/provider.go

Reply:
we then renamed common to shared !😁